website/content/en/v0.28/concepts/node-templates.md

```diff
@@ -38,7 +38,8 @@ status:
   subnets: { ... }               # resolved subnets
   securityGroups: { ... }        # resolved security groups
 ```
-Refer to the [Provisioner docs]({{<ref "./provisioners" >}}) for settings applicable to all providers.
+Refer to the [Provisioner docs]({{<ref "./provisioners" >}}) for settings applicable to all providers. For some example `AWSNodeTemplate` configurations, see the [examples here](https://github.com/aws/karpenter/blob/main/examples/provisioner/).
```

Comment:
This feature isn't released as part of `v0.28` so this shouldn't be included in the `v0.28` docs